packages/material-ui/src/Select/SelectInput.js

```diff
@@ -385,7 +399,7 @@ const SelectInput = React.forwardRef(function SelectInput(props, ref) {
         }
       },
       role: 'option',
-      selected,
+      selected: index === 1 && arr[0].props.value === undefined ? hasOptionSelected() : selected,
```

Comment:
Here when the first child is a ListSubheader it sets the selected true to the second option, to gains the focus when the list of options is open, with the tabindex = 0, to the arrow down works.